assets/onboarding/features/use-features-polling.js

```diff
@@ -0,0 +1,38 @@
+import { useState, useEffect } from '@wordpress/element';
+import { useSelect, useDispatch } from '@wordpress/data';
+
+/**
+ * Features polling hook.
+ *
+ * @param {boolean} active Flag whether polling is active.
+ */
+const useFeaturesPolling = ( active ) => {
+	const [ pollingCount, setPollingCount ] = useState( 0 );
+
+	const features = useSelect(
+		( select ) =>
+			select( 'sensei/setup-wizard' ).getStepData( 'features', true ),
+		[ pollingCount ]
+	);
+	const { invalidateResolution } = useDispatch( 'sensei/setup-wizard' );
+
+	useEffect( () => {
+		if ( ! active ) {
+			return;
+		}
+
+		const timer = setTimeout( () => {
+			// Invalidate resolution to get fresh content from the server.
+			invalidateResolution( 'getStepData', [ 'features', true ] );
+			setPollingCount( ( n ) => n + 1 );
+		}, 5000 );
```

Comment:
It feels like we can decrease this to around 1-2 seconds, so plugins are ticked one by one. When testing 5 seconds was enough for two of the plugins to be installed, and marking them successful at the same time feels a bit clunky.

Reply:
Updated to 2 seconds here: https://github.com/Automattic/sensei/pull/3190/commits/56f62058d8d202038b58e4e37610c24b6197a7aa